Tests/StreamChatUITests/Helpers.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    /// Simulate embedding in superview to run view lifecycle methods.
+    func embedInSuperview() {
+        UIView().addSubview(self)
+    }
```

Comment:
Maybe we should call this `simulateAddingToSuperview()` or even `executeLifecycleMethods()`?